asyncscope.md

```diff
@@ -2393,7 +2389,41 @@ creates an asynchronous operation (__[async.ops]__) that, when started:
 
 ## `execution::spawn`
 
-spec here
+Add the following as a new subsection immediately after __[exec.nest]__:
+
+::: add
+__`std::execution::spawn` [exec.scope.spawn]__
+
+[1]{.pnum} `spawn` associates the given input sender with the given token's async scope and, on success, eagerly starts the input sender.
+
+[2]{.pnum} The name `spawn` denotes a customization point object. For subexpressions `sndr`, `token`, and `env`, let `Sndr` be 
+`decltype((sndr))`, let `Token` be `decltype((token)), and let `Env` be `decltype((env))`. If `sender<Sndr>` or `async_scope_token<Snder>` is false, 
```

Comment:
I think the formatting is a bit wonky because there's a missing backtick here.
```suggestion
`decltype((sndr))`, let `Token` be `decltype((token))`, and let `Env` be `decltype((env))`. If `sender<Sndr>` or `async_scope_token<Snder>` is false, 
```